File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dict={}
        for i,ele in enumerate(nums):
            k=target-ele
            if k in dict:
                return [dict[k],i]
            else:
                dict[ele]=i
        return []
        

File: 1-two-sum/1-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        //Using HashMap as we have to return Index
        HashMap<Integer,Integer> map=new HashMap();
        int element=0;
        for(int i=0;i<nums.length;i++){
            element=target-nums[i];
            if(map.containsKey(element)){
                return new int[]{map.get(element),i};
            }
            map.put(nums[i],i);
        }
        return new int[]{0,0};
        
    }
}